Content.Server/Cargo/Systems/CargoSystem.Orders.cs

```diff
@@ -130,6 +131,20 @@ private void OnApproveOrderMessage(EntityUid uid, CargoOrderConsoleComponent com
                 return;
             }
 
+            // goob edit - resource siphon blocking access
+            var eqe = EntityQueryEnumerator<ResourceSiphonComponent>();
```

Comment:
```suggestion
            var eqe = EntityQuery<ResourceSiphonComponent>();
```
If you dont need the entityUid just use EntityQuery()

Reply:
i need